support_for_advertising_use_cases.md

```diff
@@ -165,19 +190,87 @@ These use-cases are more focused on the publisher's perspective.
 |[On-site Sponsored Product](#on-site-sponsored-product)| There should not be any conflict with Chrome's proposed “[Privacy Model for the Web](https://github.com/michaelkleber/privacy-model)” | Similar answer as that for Chrome. This should not be in conflict with Webkit's Tracking Prevention Policy given the measurement is entirely within the scope of a single publisher website. | |
-| [Search](#search) | unclear, but relies a lot on contextual | unclear | |
+|[Search](#search) | unclear, but relies a lot on contextual | unclear | |
+| [Ad Safety](#ad-safety) | Very limited support as opaque iframe aggregated reporting allow for very little ex post audit. | No Support |  |
+| [Advertisers exclusion](#advertisers-exclusion) | Unclear. To be defined. |  |  |
+| [Revenue management](#revenue-management) | Little support in TURTLEDOVE, as nor reporting nor updating bidding JavaScripts is not real-time. This means delays in both information and action, leading to very complex and inaccurate budget/revenue management. |  |  |
 
 # User Needs
 
-# Current Proposals and Browser Support
+- [Why am I seeing this ad?](#why-am-i-seeing-this-ad)
+- [Opt-out of an advertiser, category or product](#opt-out-of-an-advertiser-category-or-product)
+- [Opt-out of a marketing service](#opt-out-of-a-marketing-service)
+- [Seeing ads relevant to me](#seeing-ads-relevant-to-me)
+- [Smooth user experience](#smooth-user-experience)
 
 | Use-case | Chrome | Safari | Community Proposals |
 |----------|--------|--------|---------------------|
-| [Exclusion Targeting](#exclusion-targeting) | Acknowledgement that this is a valuable use-case and a link to Facebook’s PETREL proposal on this [GitHub Issue](https://github.com/michaelkleber/turtledove/issues/3) | No support | Facebook proposal for “[Private Exclusion Targeting Rendered Exclusively Locally (PETREL)](https://github.com/w3c/web-advertising/blob/master/PETREL.md)” |
-| [Frequency Capping](#frequency-capping) / [Frequency Optimization](#frequency-optimization) | For ads served via TURTLEDOVE interest-group targeting, the proposal offers a way to [handle frequency capping on-device](https://github.com/michaelkleber/turtledove#on-device-auction).  For other types of targeting, while it will not be possible to enforce a hard "frequency-cap" across multiple websites, it might be possible to calibrate a target average frequency model. See discussion about how to do this on the explainer for the [Aggregate Reporting API](https://github.com/csharrison/aggregate-reporting-api#advanced-example-calibrating-a-frequency-capping-model). | No support | |
```

Comment:
Similar feedback here. We get feedback from people that it's annoying to see the same ad over and over again. It seems like frequency capping is a valuable use-case from the perspective of people.

Reply:
Idem.
This use-case is already in Advertiser Needs. I thought there was no need for duplication, even though I understand that this use-case has implications for the user as well.